### ISSUE TITLE
Error: 429: Request too fast, please try again later

### ISSUE THREAD

**Issue by tosinabase** [LuKks/debank-re#1]
The library worked well for me a few weeks ago, but now I'm encountering the following error:
```
/home/fahil/projects/node_modules/debank-re/index.js:75
      throw new Error(body.error_code + ': ' + body.error_msg)
            ^

Error: 429: Request too fast, please try again later.
    at Debank.api (/home/fahil/projects/node_modules/debank-re/index.js:75:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async test (/home/fahil/projects/debank_run_user_info.js:19:18)

Node.js v20.16.0

```

While using:
```js
const Debank = require('debank-re')
const wallet = "some wallet address"
const debank = new Debank()
const user = await debank.get('/user', { id: wallet })
```
I obtain the same error even if I make just one request from a new server and when I use different proxies.

**Comment by LuKks** [comment]
Thanks for the heads up. Fixed in https://github.com/LuKks/debank-re/commit/0f6685080e13fa9b9b514f09252e6e81621419cf v1.1.1

**Comment by imacrosid** [comment]
> Thanks for the heads up. Fixed in [0f66850](https://github.com/LuKks/debank-re/commit/0f6685080e13fa9b9b514f09252e6e81621419cf) v1.1.1

```
2162/5242 0x565e752..F7CF1968c | 0.00 USD
2163/5242 0xD8147c7..5D5eD8B4 | 0.00 USD
L:\Learning\Crypto\Others\debank-balance-checker\node_modules\debank-re\index.js:76
      throw new Error(body.error_code + ': ' + body.error_msg)
            ^

Error: 429: Request too fast, blocked by waf
    at Debank.api (L:\Learning\Crypto\Others\debank-balance-checker\node_modules\debank-re\index.js:76:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main (file:///L:/Learning/Crypto/Others/debank-balance-checker/index.js:32:19)

Node.js v22.6.0
```
Still facing issue request too fast but now blocked by waf
```
$ npm list debank-re
debank-balance-checker@1.0.0 L:\Learning\Crypto\Others\debank-balance-checker
└── debank-re@1.1.1
```

**Comment by LuKks** [comment]
@imacrosid Are you still having the same issue? Tests are passing for me.

You're actually making too many requests probably, you can pass options like `{ proxyUrl }`

Read `got` or `got-scrapping` documentation to learn more about that, but it's as simple as passing a proxy URL.
